docs/users.rst

```diff
@@ -529,6 +529,9 @@ add_timestamp_to_release
   the container image will be available in container registry at:
   ``my-container-registry.example.com/fedora/rsyslog:32-5.20191007151825``.
 
+ignore_isolated_builds
+  Boolean to control whether to rebuild when parent image triggering build was isolated build.
```

Comment:
Could we have some context on this one? We could explain when this would be useful for a user

Reply:
well it is solely on user if he finds it useful, and I wouldn't like to make it more complicated as it is really simple option